policies/ucs_domain_policies/flow_control_policy/python/flow_control_policy.py

```diff
@@ -16,10 +16,19 @@
 
 
 def create_organization():
```

Comment:
Can we make the function name get_organization(name) and set name = "default" which user can changes based on their preferred name